include/oneapi/tbb/detail/_machine.h

```diff
@@ -25,14 +25,19 @@
 #include <cstdint>
 #include <cstddef>
 
-#ifdef _MSC_VER
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
 #include <intrin.h>
+#include <windows.h> // SwitchToThread()
```

Comment:
`detail/_machine.h` is public header file and inclusion `windows.h` with specific macros (`WIN32_LEAN_AND_MEAN`) into user appication is not desirable. Include it under `__TBBMALLOC_BUILD` guard because it is defined only for tbbmalloc build that is not typical user environment.

Reply:
Got it, I'll fix soon